.gitignore

```diff
@@ -1,5 +1,5 @@
 *.swp
-*.native
-*.byte
-*.install
 _build
+syslog-message.install
+src/.merlin
+test/.merlin
```

Comment:
I think it makes sense to just ignore all `.merlin` files

Reply:
Sure, I have little opinion on that!